README.md

```diff
@@ -456,9 +456,61 @@ Defaults:
   --external-modules-download-path:.external_modules
   --evaluate-variables:True
 ```
+
+### Pre-commit hook
```

Comment:
please remove it from the docs here and extend the docs here https://github.com/bridgecrewio/checkov/blob/main/docs/4.Integrations/pre-commit.md

Reply:
Updated documentation.